cn_indian_payroll/cn_indian_payroll/overrides/payroll_configuration.py

```
@@ -1,119 +1,343 @@
 
+                for field in custom_field:
 
+                    custom_field_check = frappe.get_list('Custom Field',
+                    filters={
+      
+                        "fieldname":field.get('field_name'),
+                        "dt":"Salary Structure Assignment",
 
+                        
+                        },
+                        fields=['*']
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)